app/assets/javascripts/admin/spree/orders/variant_autocomplete.js.erb

```diff
@@ -154,3 +157,20 @@ addVariantFromStockLocation = function() {
   }
   return 1
 }
+
+showAlert = function(message) {
+  $('#custom-alert .message').text(message)
+  $('#custom-alert .confirm').click(function(e) {
+    $('#custom-alert').hide()
+  })
```

Comment:
Whenever `showAlert` is called, a new hide-callback is added to this element. That could become a bit inefficient after a while. This callback should probably be added only once, in some initialising code.